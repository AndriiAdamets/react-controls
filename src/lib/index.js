import Button from './components/Button';
import Card from './components/Card';
import Tooltip from './components/Tooltip';
import Checkbox from './components/Checkbox';
import Input from './components/Input';
export { Button, Card, Tooltip, Checkbox, Input };