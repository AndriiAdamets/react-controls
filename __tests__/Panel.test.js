import React from 'react';
import renderer from 'react-test-renderer';
import { Card } from '../src/lib';

test('Panel', () => {
  const component = renderer.create(
    <Card>
      <Card.Header>
        <Card.Title>
          Card Title
        </Card.Title>
      </Card.Header>
      <Card.Body>
        Card Body
      </Card.Body>
    </Card>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});