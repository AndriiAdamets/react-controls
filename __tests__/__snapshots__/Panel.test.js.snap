// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel 1`] = `
<section
  className="wrc-card"
>
  <div
    className="wrc-card__header"
  >
    <h3
      className="wrc-card__title"
    >
      Card Title
    </h3>
  </div>
  <div
    className="wrc-card__body"
  >
    Card Body
  </div>
</section>
`;
